Tools/scripts/deepfreeze.py

```diff
@@ -20,6 +20,9 @@
 verbose = False
 identifiers, strings = get_identifiers_and_strings()
 
+# This must be kept in sync with opcode.py
+RESUME = 151
+
```

Comment:
Can't this be ``opcode.opmap["RESUME"]``?

Reply:
No, that would introduce a dependency on the latest opcode.py which would mean you cannot run deepfreeze.py with e.g. Python 3.10. On Windows and for cross compilations we need to be able to do that. (Windows doesn't have a separate bootstrap Python because it would near-doubel build times.)

We started with this (see https://github.com/python/cpython/issues/93771).